examples/shallow_water/linear_thermal_galewsky_jet.py

```diff
@@ -0,0 +1,164 @@
+from gusto import *
```

Comment:
I realise this example has just been added here for now and maybe you're still planning to do this, but we will need to make it match the format of the other examples to ensure that it gets run through the testing -- i.e. it will need a `__main__` method and use arg-parse so that it can be called from the command line with different arguments.

To be consistent with everything else, we'd also need a plotting script and a figure, although I think we could set up an issue and do that in a different PR

Reply:
I did just put this here for now but I can make those changes if we decide we want this example to stay.

Reply:
I think it would be a nice example to have -- I think we should have something with the linear thermal equations